sky/clouds/aws.py

```diff
@@ -289,78 +305,79 @@ def _make(instance_list):
     def check_credentials(self) -> Tuple[bool, Optional[str]]:
         """Checks if the user has access credentials to this cloud."""
         try:
-            # pylint: disable=unused-import
+            # pylint: disable=top-level-import-outside-toplevel,unused-import
             import boto3
             import botocore
         except ImportError:
             raise ImportError('Fail to import dependencies for AWS.'
                               'Try pip install "skypilot[aws]"') from None
-        help_str = (
-            ' Run the following commands:'
-            '\n      $ pip install boto3'
-            '\n      $ aws configure'
-            '\n    For more info: '
-            'https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html'  # pylint: disable=line-too-long
-        )
         # This file is required because it will be synced to remote VMs for
         # `aws` to access private storage buckets.
         # `aws configure list` does not guarantee this file exists.
         if not os.path.isfile(os.path.expanduser('~/.aws/credentials')):
-            return (False, '~/.aws/credentials does not exist.' + help_str)
+            return (False, '~/.aws/credentials does not exist. ' +
+                    self._STATIC_CREDENTIAL_HELP_STR)
 
         # Checks if the AWS CLI is installed properly
         try:
             _run_output('aws configure list')
         except subprocess.CalledProcessError:
             return False, (
                 'AWS CLI is not installed properly.'
-                ' Run the following commands under sky folder:'
-                # TODO(zhwu): after we publish sky to PyPI,
-                # change this to `pip install sky[aws]`
+                ' Run the following commands in the SkyPilot codebase:'
                 '\n     $ pip install .[aws]'
```

Comment:
Can this be changed to `pip install "skypilot[aws]"` now?

Reply:
Should we do it when we publish the new release including these changes? Added a TODO here to change the installation command. Wdyt?